libs/utils/decorate.js

```diff
@@ -278,3 +278,23 @@ export function applyInViewPortPlay(video) {
     observer.observe(video);
   }
 }
+
+export function decorateMultiViewport(el) {
+  const viewports = [
+    '(max-width: 599px)',
+    '(min-width: 600px) and (max-width: 1199px)',
+    '(min-width: 1200px)',
+  ];
+  const foreground = el.querySelector('.foreground');
+  if (foreground.childElementCount === 2 || foreground.childElementCount === 3) {
+    [...foreground.children].forEach((child, index) => {
+      const mq = window.matchMedia(viewports[index]);
+      const setContent = ({ matches }) => {
+        if (matches) foreground.replaceChildren(child);
+      };
+      setContent(mq);
+      mq.addEventListener('change', setContent);
```

Comment:
Since this is nicely grouped together, do you need to pass the arguments around? On L293, could we not just simply check for `if (mq.matches)`?